images/prometheus-config-reloader/tests/main.tf

```diff
@@ -38,9 +38,24 @@ resource "helm_release" "kube-prometheus-stack" {
   }
 }
 
-module "helm_cleanup" {
+data "oci_exec_test" "helm_cleanup" {
+  digest = var.digest
```

Comment:
there should be _depends_on_ section to wait until the helm command succeed I guess to provide the dependency between resources.

Reply:
Thats a valid suggestion @developer-guy, thanks for that, will include and push up